pom.xml

```diff
@@ -165,6 +165,12 @@
       <version>4.29.2</version>
     </dependency>
 
+  
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-lang3</artifactId>
+      <version>3.9</version>
+    </dependency>
```

Comment:
This appears to be to support:

```
import org.apache.commons.lang3.time.DateFormatUtils;
```

But it wasn't clear if you are actually using DateFormatUtils anywhere in this code.

So could you remove this change to the pom.xml as well as the unnecessary import?